python/mkdocs/docs/configuration.md

```diff
@@ -73,7 +73,22 @@ For the FileIO there are several configuration options available:
 | adlfs.sas-token         | NuHOuuzdQN7VRM%2FOpOeqBlawRCA845IY05h9eu1Yte4%3D                                          | The shared access signature                                                                                                                                                                                                                                                          |
 | adlfs.tenant-id         | ad667be4-b811-11ed-afa1-0242ac120002                                                      | The tenant-id                                                                                                                                                                                                                                                                        |
 | adlfs.client-id         | ad667be4-b811-11ed-afa1-0242ac120002                                                      | The client-id                                                                                                                                                                                                                                                                        |
-| adlfs.client-secret     | oCA3R6P\*ka#oa1Sms2J74z...                                                                 | The client-secret                                                                                                                                                                                                                                                                    |
+| adlfs.client-secret     | oCA3R6P\*ka#oa1Sms2J74z...                                                                | The client-secret                                                                                                                                                                                                                                                                    |
+
+### Google Cloud Storage
```

Comment:
It would be nice if the Python GCS properties could be consistent with the Java properties where possible. Java already has some properties defined so starting with those would help avoid any backwards compatibility issues (rather than changing them). There is also a [PR](https://github.com/apache/iceberg/pull/8073) to add OAuth2 access token properties on the Java side.